config/public.yml

```diff
@@ -0,0 +1,21 @@
+default:
+    project_desc: The William F. Cody Archive
+    solr_core: api_cody
+#   es_type: cody_archive
+    shortname: cody_archive
+    vra_solr_xsl: collections/data_cody/scripts/solr_transform_vra.xsl
+    tei_solr_xsl: collections/data_cody/scripts/solr_transform_tei.xsl
+    xsl_params:
```

Comment:
I think this should actually be named variables_solr:  https://github.com/CDRH/data/blob/es/config/public.yml#L49

That might be why the slug is not populating?